internal/pkg/argocdapp/argocdapp.go

```diff
@@ -42,3 +42,13 @@ func kubectlApply(file string) {
 	}
 	log.Println(string(strings.TrimSuffix(string(stdout), "\n")))
 }
+
+func kubectlDelete(file string) {
+	cmd := exec.Command("kubectl", "delete", "-f", file)
+	stdout, err := cmd.Output()
+	if err != nil {
+		fmt.Println(err.Error())
+		return
+	}
+	log.Println(string(strings.TrimSuffix(string(stdout), "\n")))
```

Comment:
```
log.Println(strings.TrimSuffix(string(stdout), "\n")) 
```
is fine. 

The **string** inside `log.Println(**string**(strings.TrimSuffix(string(stdout), "\n")))` is redundant.

Please help to fix the same issue at `kubectlApply()` function.

Reply:
log info removed